sprites.py

```diff
@@ -1,25 +1,25 @@
 # -*- coding: utf-8 -*-
 
-#Copyright (c) 2007-8, Playful Invention Company.
```

Comment:
For changes to `sprites.py` it's been maintained in the [turtleart activity repo](https://github.com/sugarlabs/turtleart-activity/blob/master/TurtleArt/sprites.py) so you might want to update this one with that as changes here might've been made there already or if that's not the case then could you also make those changes that exist here but not there to that file?